src/main/java/org/jenkinsci/test/acceptance/po/TopLevelItem.java

```diff
@@ -58,8 +58,7 @@ public TopLevelItem(PageObject context, URL url, String name) {
     }
 
     /**
-     * Renames the job. Opens the configuration section, sets the name and saves the form. Finally the rename is
-     * confirmed.
+     * Changes the description. Opens the configuration section, sets the description and saves the form.
```

Comment:
Well, I mean in a class that is not yours ;-) your method is the first one without verb in the Job class. But anyway, since we (sadly) don't have a coding style in this project field free to keep it as it is. 
 
BTW: GithubBranchSource is even worse: there is a public field *and* an method with the same name: which one should I choose?